specification/app/resource-manager/Microsoft.App/preview/2025-02-02-preview/examples/ManagedEnvironmentsStorages_CreateOrUpdate.json

```diff
@@ -31,6 +31,24 @@
         }
       }
     }
-  }
+  },
+  "201": {
+      "body": {
+        "id": "/subscriptions/8efdecc5-919e-44eb-b179-915dca89ebf9/resourceGroups/examplerg/providers/Microsoft.App/managedEnvironments/managedEnv/storages/jlaw-demo1",
+        "name": "jlaw-demo1",
+        "type": "Microsoft.App/managedEnvironments/storages",
+        "properties": {
+          "azureFile": {
+            "accountName": "account1",
+            "shareName": "share1",
+            "accessMode": "ReadOnly",
+            "accountKeyVaultProperties": {
```

Comment:
Should  we update this for other examples like 200 response

Reply:
Update this file to only include 201, and Nfs files to only include 200.